src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidation.java

```diff
@@ -19,29 +20,42 @@
 
     private final OfficerService officerService;
 
+    private Boolean officerValidationFlag;
+
     @Autowired
-    public CompanyOfficerValidation(OfficerService officerService){
+    public CompanyOfficerValidation(OfficerService officerService, Boolean officerValidationFlag){
         this.officerService = officerService;
+        this.officerValidationFlag = officerValidationFlag;
     }
 
     @Override
     public void validate(CompanyProfileApi companyProfileApi) throws EligibilityException, ServiceException {
         LOGGER.info("Validating Company Officers for: {}", companyProfileApi.getCompanyNumber());
+        if (!Boolean.TRUE.equals(officerValidationFlag)) {
+            LOGGER.debug("OFFICER VALIDATION FEATURE FLAG off skipping validation");
+            return;
+        }
         var officers = officerService.getOfficers(companyProfileApi.getCompanyNumber());
-        var officerCount = getOfficerCount(officers.getItems(), officers.getActiveCount());
+        var officerCount = getOfficerCount(officers.getItems());
         if (officerCount != null && officerCount > 1) {
             LOGGER.info("Company Officers validation failed for: {}", companyProfileApi.getCompanyNumber());
             throw new EligibilityException(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_MORE_THAN_ONE_OFFICER);
         }
         LOGGER.info("Company Officers validation passed for: {}", companyProfileApi.getCompanyNumber());
     }
 
-    public Long getOfficerCount(List<CompanyOfficerApi> officers, Long activeCount) {
-        for(CompanyOfficerApi i: officers) {
-            if (i.getOfficerRole().getOfficerRole().contains("secretary")) {
-                activeCount--;
+    public Long getOfficerCount(List<CompanyOfficerApi> officers) {
```

Comment:
I would return a primitive `int` instead of an Object `Long` from this method.
You only need the object if null is a possibility which in this method is not.
And the difference between int and long is the maximum value.
And int can cope with up to 2147483647. Which is more officers that any company could reasonably have